CHANGELOG.next.toml

```diff
@@ -22,3 +22,9 @@ message = "The MSRV has been increase to 1.74.1"
 references = ["smithy-rs#3410"]
 meta = { "bug" = false, "breaking" = false, tada = false }
 author = "rcoh"
+
+[[smithy-rs]]
+message = "Implement std::error::Error for ConstraintViolation"
```

Comment:
```sugestion
+message = "Implement `std::error::Error` for `ConstraintViolation`"
```
Why don't we implement `std::error::Error` for all operation error types, not only `ConstraintViolation`s?

I guess the reasoning is there's not an obvious `Display` representation for an arbitrary `@error` structure, but for `ConstraintViolation` there is? I don't like this asymmetry.

Reply:
We are already generating code to implement `std::fmt::Display` and `std::error::Error` on all `@error` structures.